### ISSUE TITLE
Run on Windows

### ISSUE THREAD

**Issue by metheofanis** [sanchit-gandhi/whisper-jax#62]
I've failed miserably to run it on windows.
Are there any binaries for windows?
I'm very novice at running and debugging all those CLI commands. 
Thanks.

**Comment by nikshepsvn** [comment]
windows support is absolutely abmyssal, way too hard to get running

**Comment by sanchit-gandhi** [comment]
Hey @metheofanis, @nikshepsvn - could you try opening an issue in the JAX repo? IMO the more interest they see around running JAX on Windows, the more likely they are to invest time supporting it!

Happy to answer any questions regarding Whisper JAX here.

**Comment by metheofanis** [comment]
Hey @sanchit-gandhi, I've managed to run it in Windows Subsystem for Linux.
My GPU is 3060 12GB RAM.
The performance is ~1X for the first audio and ~3X for the second!
Is that normal? Did I miss something?
I guess the advertised 70X speed is for cloud GPU and TPU setups?
Thanks!

**Comment by aardvarkoncomputer** [comment]
> 

When you run it, is it utilizing the GPU in performance monitor? I had a similar issue as I couldn't get CUDA to behave, wasn't able to resolve it.

**Comment by metheofanis** [comment]
@aardvarkoncomputer Maybe you are missing this:
https://docs.nvidia.com/cuda/wsl-user-guide/index.html

In step 3, option 1 there is a link to:
https://developer.nvidia.com/cuda-downloads?target_os=Linux&target_arch=x86_64&Distribution=WSL-Ubuntu&target_version=2.0&target_type=deb_local

After following these, it worked from GPU!

**Comment by hsking1964** [comment]
Yes, I am interested in running it on Windows 10/11 as well, but have not had any success with the installation instructions. I am not averse to learning how it works, but I am also happy to install it without knowing how it works (as long as it works) :) so a set of commands for copy and pasting at the CL or even a Windows bat file is good for me. Thanks! (The set of commands would need to be from a clean Windows install, so command to install git if necessary, etc.)

**Comment by audioscavenger** [comment]
I too would love to see some native support for Windows like the folks of whisper-faster and faster-whisper-xxl provide. thank you!
